stripper/ze_yurilnier_beta007_hdr.cfg

```diff
@@ -0,0 +1,17 @@
+;If zombies are the first to press a door button, kill off all CT players to prevent people from hiding, kind of like ze_journey
+;Don't worry about hiders as long as a group of CTs is still pressing door buttons, as it means the hiders aren't delaying the time yet
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "/lvl\d_btn\d+/"
+	}
+	insert:
+	{
+		"OnPressed" "!selfRunscriptcodeforeach (f, _ in {FireUser1=0});01"
```

Comment:
?

Reply:
If that line is there, it works as intended. If that line is removed, it spits out this error:
`CALLSTACK
*FUNCTION [main()] InputRunScript line [1]

LOCALS
[@ITERATOR@] 2
[_] 0
[f] "FireUser1"
[this] TABLE`
I don't know why, but magically that line fixed it in the same way it magically fixed me attempting to do it in the dark souls stripper as well: https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/8438a55bd7c19dd95156c11c26ffd31054d01c7e/stripper/ze_dark_souls_ptd_csgo2.cfg#L79

Reply:
You missed the first line of this error, which gives a lot more context.
```
AN ERROR HAS OCCURED [the index 'activator' does not exist]
CALLSTACK
*FUNCTION [main()] InputRunScript line [1]
LOCALS
[@ITERATOR@] 2
[_] 0
[f] "FireUser1"
[this] TABLE
```
Though it still doesn't make sense why this is happening.